packages/did-core-test-server/suites/did-spec/default.json

```diff
@@ -2,53 +2,59 @@
   {
     "name": "did-spec",
     "didMethods": {
-      "did:example": {
+      "did:key": {
         "supportedContentTypes": [
           "application/did+json",
           "application/did+ld+json"
         ],
-        "dids": ["did:example:123"],
-        "didParameters": {
-          "hl": "did:example:123?hl=zQmWvQxTqbG2Z9HPJgG57jjwR154cKhbtJenbyYTWkjgF3e",
-          "service": "did:example:123?service=agent&relativeRef=%2Fpath%2Fto%2Fresource",
-          "relative-ref": "did:example:123?service=agent&relativeRef=%2Fpath%2Fto%2Fresource",
-          "version-id": "did:example:123?version-id=0.1.0",
-          "version-time": "did:example:123?version-time=1601151242"
-        },
-        "did:example:123": {
+        "dids": ["did:key:z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB"],
+        "did:key:z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB": {
           "application/did+json": {
             "didDocument": {
-              "@context": ["https://www.w3.org/ns/did/v1"],
-              "id": "did:example:123",
-              "canonicalId": "did:example:one-two-three",
-              "equivalentId": "did:example:one-two-three",
+              "@context": [
+                "https://www.w3.org/ns/did/v1",
+                {
+                  "@base": "did:key:z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB"
+                }
```

Comment:
This makes me sad -- feels like something the did-key library implementation should do by setting the base instead of it being hard coded in the `@context`.

Reply:
It is extraneous information most of the time... but experience with JSON-LD has taught us that it is better to be explicit about this sort of thing.